docs/operations/figures/k8s-ces-overview-components_de.puml

```diff
@@ -0,0 +1,100 @@
+@startuml
+!define CLOUDOGUURL https://raw.githubusercontent.com/cloudogu/plantuml-cloudogu-sprites/master
+
+!includeurl CLOUDOGUURL/common.puml
+!includeurl CLOUDOGUURL/dogus/cloudogu.puml
+!includeurl CLOUDOGUURL/dogus/scm.puml
+!includeurl CLOUDOGUURL/dogus/redmine.puml
+!includeurl CLOUDOGUURL/dogus/nginx.puml
+!includeurl CLOUDOGUURL/dogus/postgresql.puml
+!includeurl CLOUDOGUURL/tools/k8s.puml
+!define SECONDARY_COLOR #55EE55
+
+rectangle "Cluster" as cluster <<$k8s>> {
+  rectangle k8sapi as "K8s API-Server" <<$k8s>> #white
+  rectangle k8sres as "*"
+
+  rectangle "Namespace" as namespace <<$k8s>> #white {
+    rectangle mgrComps as "K8s-Komponenten" #white {
+
+      rectangle servdisc as "k8s-service-discovery" <<$cloudogu>>
+      note bottom of servdisc #ffffee
+        erzeugt Warp-Menü
```

Comment:
The service discovery also creates the ingress resources and starts/stops the maintenance mode

Reply:
Ay, thanks, it seems there's a lot I forgot :+1:  Will add

- [ ] done